website/docs/type-assertions.md

```diff
@@ -100,9 +100,15 @@ will still be caught as a missing method statically.
 
 ## `T.must`
 
+<a id="tmust_because"></a>
+
 `T.must` is for asserting that a value of a [nilable type](nilable-types.md) is
 not `nil`. `T.must` is similar to `T.cast` in that it will not necessarily
 trigger an error when `srb tc` is run, but can trigger an error during runtime.
+
+`T.must_because` is like `T.must` but also takes a reason why the value is not
```

Comment:
Maybe also: "The provided reason is present in any error messages raised." or somesuch?